apps/ett/e2e-ett/src/support/landing-page.po.ts

```diff
@@ -14,5 +14,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
It's okay to have a "copy" in the end-to-end test suite since changing a route path is a breaking change for users (breaks existing links) so it should alert us by failing our tests so that we are aware and can intentionally make a decision on whether this is what we want. If the end-to-end tests referred to a library variable, a route path change would go unnoticed as far as the end-to-end tests are concerned.

Reply:
Brilliant, makes good sense!